index.js

```diff
@@ -40,7 +40,8 @@ function DB (opts) {
     map: function (row, next) {
       if (!row.value) return null
       var v = row.value.v, d = row.value.d
-      if (v && v.lat !== undefined && v.lon !== undefined) {
+      var k = row.value.k
+      if (k && v.lat !== undefined && v.lon !== undefined) {
```

Comment:
Good call re comment vs function.

> One thing though, a delete could have lat and lon defined, that wouldn't affect here right?

Right. Deletions have a `d` key, and insertions have a `k` key. Since this new code checks for the presence of a `k`, even deletions with values would be ignored.